lab_1/main.py

```diff
@@ -1,94 +1,121 @@
-"""
-Lab 1
-A concordance extraction
-"""
+import math
 
 
+def checker(con_object, type_ob):
+    if type_ob is list and type(con_object) is list:
```

Comment:
There is a short-hand version of this check:
`isinstance(object, type)` - take a look at differences.

Reply:
Used, but it does not exclude the process of checking the type the func gets. Thanks